client/data/account-keys/__tests__/actions.test.js

```diff
@@ -63,9 +61,6 @@ describe( 'Account keys actions tests', () => {
 					} ),
 				] )
 			);
-			expect(
```

Comment:
This test had to be removed because we are now using the result of apiFetch to control the execution flow in the tested function. 
In the tests, as we call `updateAccountKeysValues` using the spread operator, the value from the call to apiFetch is not maintained inside the function to be used for the accountData variable.

We did some brainstorming yesterday and couldn't find a way to mock apiFetch the same way it is used in the wordpress-data controls.